src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -1140,6 +1170,33 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal sealed partial class CompoundAssignmentOperation : BaseCompoundAssignmentExpression
+    {
+        public CompoundAssignmentOperation(IOperation target, IOperation value, IConvertibleConversion inConversionConvertible, IConvertibleConversion outConversionConvertible, BinaryOperatorKind operatorKind, bool isLifted, bool isChecked, IMethodSymbol operatorMethod, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) : base(inConversionConvertible, outConversionConvertible, operatorKind, isLifted, isChecked, operatorMethod, semanticModel, syntax, type, constantValue, isImplicit)
+        {
+            TargetImpl = target;
+            ValueImpl = value;
+        }
+
+        protected override IOperation TargetImpl { get; }
```

Comment:
Where do we override `Children` and return the `Target` and `Value` child nodes?

Reply:
Ah, I see we do it in `AssignmentExpression`